uwblueprint/planet-read
# Create Review Page and seperate translation table

## Notion ticket link

[Add review page](https://www.notion.so/uwblueprintexecs/a658c933d18c48d4b43fd99bda2bd3b4?v=4a47edc475b34bb99b25ff1fae2a5507&p=777817bbffbb4b268186155c14149f2a)



## Implementation description
* modularize translation table to `TranslationTable` component
  * add disable option
  * display `status` badge an useless comment button
* create review page




## Steps to test
1. login
2. visit translation page 
3. make sure translation page still works
4. There are currently no auth checks on the /review page, so poke around there too



## What should reviewers focus on?
* thoughts on where to save types shared between review and translate pages? eg `Content` (currently redeclared) and `StoryLine` (currently in `AutoSave`) 
* need to include `status` in `StoryTranslationContentDTO` on a different pr. statuses are just `Default` for now

## Checklist
